deploy/inventory/sandbox.yml

```diff
@@ -0,0 +1,29 @@
+---
+# The difference between this environment and aria-at-staging.w3.org is that
+# this environment is Bocoup-internal, and used for testing and sharing the
+# more drastic or long-term changes we're making.
+#
+# - The sandbox environment will track the dev branch, while staging and
```

Comment:
Good idea to add this documentation!

Maybe change to "develop" branch since that's the literal branch name?

Reply:
Sure. I copied this text from [Bocoup's internal design documentation](https://docs.google.com/document/d/1DEDMgrKim7ABg7gIbZ9ZGc6ITkVU_bXnJS6KuzmaVVw/edit), so you may want to update that as well.

Reply:
Good idea - I updated it there too.